run_motion_transfer.py

```
@@ -0,0 +1,100 @@
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'packages', 'providers', 'fal', 'avatar-generation'))
+
+from dotenv import load_dotenv
+load_dotenv()
+
+import fal_client
+from fal_avatar import FALAvatarGenerator
+
+def download_video(url: str, output_path: str) -> str:
+    """Download video from URL to local path."""
+    print(f"📥 Downloading video from: {url}")
+    response = requests.get(url, stream=True)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)